.github/ISSUE_TEMPLATE/bug-report--quasar-v2.yml

```diff
@@ -0,0 +1,139 @@
+name: Bug report (Quasar v2)
+description: Create a bug report to help us improve
+labels: ['kind/bug 🐞', '🔝 Qv2']
+
+# Note: Do NOT use commas(,) in options with 'multiple: true', the issue processing workflow won't be able to label it
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Read Before Proceeding Further
+
+        This form is for submitting a **bug report** for Quasar Framework(**v2**).
+        So, if you are using another version, just seeking help, or want to submit a feature request, please return back and choose a more suitable option from the list. 
+
+        Also, please search your issue on the repo and make sure it does not already exist.
+
+        Please fill out the form correctly according to the instructions, try to be as clear as possible, and provide as many details as possible, for our team to better understand the issue and resolve it faster.
+        Issues with an unclear description or lacking information have a lower priority. They may even get closed without taking action.
+
+        Please also give a read to [Reporting an Issue | Contribution Guide](https://quasar.dev/contribution-guide/contribution-guide#reporting-an-issue) section of our docs for a more detailed explanation.
+
+        Thanks for your understanding, we truly appreciate your effort to fill out this form!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Clearly and concisely explain the problem.
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: What did you expect to happen?
+    validations:
+      required: true
+  - type: input
+    id: repro-url
+    attributes:
+      label: Reproduction URL
+      description: |
+        Fork a [Codepen](https://codepen.quasar.dev) or a [jsFiddle](https://jsfiddle.quasar.dev) or a [Codesandbox](https://codesandbox.quasar.dev) and hit save then copy-paste the URL here.
+        _You can provide a GitHub repo only as a last resort, keep in mind that it will be prioritized less as it makes the repro process slower._
```

Comment:
```suggestion
        _You can provide a GitHub repo only as a last resort. Keep in mind that it will be prioritized less as it makes the reproduction process slower._
```